### PR TITLE
Update README.md - do not clone it, make a fork

### PR DESCRIPTION
**Pull request by kandl**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ To successfully complete this tutorial, you are required to:
 
 ### Initialization
 
-1. Run `git clone git@github.com:gooddata/gooddata-js-interview.git`.
+1. Make a [fork of this repository and clone it](https://help.github.com/en/articles/fork-a-repo)
```
Comment (no23reason) [inline README.md]: Missing a full stop at the end of the line ;)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-interview-oss-check/4/ : SUCCESS in 6s

**Pull request merged**